tests/unit/behaviors/TranslatableCmsObjectTest.php

```diff
@@ -84,7 +84,7 @@ public function testGetTranslationValueNoFallback()
 
         $this->assertEquals('Awww yiss', $obj->markup);
 
-        $obj->noFallbackLocale()->translateContext('fr');
```

Comment:
Please, keep the tests for the deprecated method (until it actually gets removed).